common/app/views/fragments/items/elements/starRating.scala.html

```diff
@@ -1,5 +1,6 @@
 @(rating: Int)(implicit request: RequestHeader)
 
+<span class="u-h">@rating out of 5 stars.</span>
 <div class="stars fc-item__stars" role="presentation">
```

Comment:
do you think this should be `aria-hidden="true"` instead of `role="presentation"`?